main.go

```diff
@@ -172,6 +173,16 @@ func _main(ctx context.Context) error {
 			DBName:       cfg.PGDBName,
 			QueryTimeout: cfg.PGQueryTimeout,
 		}
+		// Replace {netname} with curnet
+		if strings.Contains(dbi.DBName, "{netname}") {
+		  reg, err := regexp.Compile("[^a-zA-Z]+")
+		  if err != nil {
+		    return err
+		  }
+		  
+		  var networkName = reg.ReplaceAllString(curnet.String(), "");
```

Comment:
The one modification to the network name that I would make is changing to all lower case.

Reply:
Actually, please see the `netName` function at the end of config.go that gets the current lowercase network name out of the active network (chaincfg).

Reply:
For the moment I updated this line to `networkname := strings.ToLower(curnet.String())`
I could add some logic to main.go and call 'netname` but that would be duplicating logic that already exists in config.go.  Alternatively I could add `cfg.NetworkName = netname(activeNet)` to config.go and eliminate this line completely.

Reply:
`netName` is in the same namespace (`package main`) as `_main`, so you can use it directly.  Unlike some other languages where the compilation unit is the source file (and whatever included headers), Go has all the source files for a package as part of one compilation unit.